### PR TITLE
[Hub Generated] Review request for Microsoft.VirtualMachineImages to add version stable/2020-02-14

### PR DESCRIPTION
**Pull request by feiyushi**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/feiyushi/azure-rest-api-specs/dev-imagebuilder-Microsoft.VirtualMachineImages-2020-02-14...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by feiyushi** [comment]
This api-version has not yet been released for SDK generation and service has not yet GA'ed, but this will be the first GA api-version. Please re-consider the breaking change.

**Review by pilor — APPROVED** [review]
From an API standpoint this version is still behind a feature flag (i.e. private preview) so I am OK with the change.

**Comment by feiyushi** [comment]
@allenjzhang, Please let me know if anything needs to be done for this PR, or please help merge. Thanks.

**Comment by feiyushi** [comment]
pushed a commit to address issue #9233.

**Comment by feiyushi** [comment]
> @allenjzhang, Please let me know if anything needs to be done for this PR, or please help merge. Thanks.

Please take a look. Thank you! @allenjzhang

File: specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2020-02-14/imagebuilder.json [hunk 1]
```diff
@@ -350,6 +347,9 @@
           }
         ],
         "responses": {
+          "200": {
+            "description": "The retrieval of the operation was successful."
```
Comment (allenjzhang) [inline specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2020-02-14/imagebuilder.json:351]: nit: Either "The operation was successful" Or more specific "The Run Operation was successful"

File: specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2020-02-14/imagebuilder.json [hunk 2]
```diff
@@ -396,6 +396,9 @@
...
```
Comment (allenjzhang) [inline specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2020-02-14/imagebuilder.json:400]: nit: Either "The operation was successful" Or more specific "The Cancel Operation was successful."

**Review by allenjzhang — APPROVED** [review]
Please update the description for better documentation. Thanks!

File: specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2020-02-14/imagebuilder.json (hunk 2 above)
Comment (feiyushi) [reply specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/stable/2020-02-14/imagebuilder.json]: updated. thx

**Pull request merged**